### ISSUE TITLE
Issue with Dedicated Logger

### ISSUE THREAD

**Issue by resech** [dtag-dev-sec/glastopf#2]
The glastopf log (glastopf.log) is showing a number of error entries anytime there's interaction with the comments section.

`2016-05-20 13:55:20,439 (glastopf.glastopf) 188.143.232.15 requested POST /utilities/comments on 63847602c5a9:80
2016-05-20 13:55:20,458 (glastopf.modules.processing.profiler) An issue occured with the dedicated logger. Please check the configuration file.
2016-05-20 13:55:20,474 (glastopf.modules.processing.profiler) An issue occured with the dedicated logger. Please check the configuration file.
2016-05-20 13:55:22,322 (glastopf.glastopf) 188.143.232.15 requested POST /utilities/comments on 63847602c5a9:80
2016-05-20 13:55:22,359 (glastopf.modules.processing.profiler) An issue occured with the dedicated logger. Please check the configuration file.
2016-05-20 13:55:22,376 (glastopf.modules.processing.profiler) An issue occured with the dedicated logger. Please check the configuration file.
`

I am still seeing comments captured in comments.txt for what it's worth. 

The issue also appears to be manifesting within the dashboard, I'm seeing entries similar to these:

`{
  "_index": "logstash-2016.05.20",
  "_type": "Glastopf",
  "_id": "AVTOcvWuVzVDZ2s_sayN",
  "_score": null,
  "_source": {
    "message": "2016-05-20 13:55:20,458 (glastopf.modules.processing.profiler) An issue occured with the dedicated logger. Please check the configuration file.",
    "@version": "1",
    "@timestamp": "2016-05-20T13:55:20.507Z",
    "path": "/data/glastopf/log/glastopf.log",
    "host": "98be59da1cac",
    "type": "Glastopf",
    "tags": [
      "_grokparsefailure"
    ]
  },
  "fields": {
    "@timestamp": [
      1463752520507
    ]
  },
  "highlight": {
    "path": [
      "/data/@kibana-highlighted-field@glastopf@/kibana-highlighted-field@/log/glastopf.log"
    ],
    "type": [
      "@kibana-highlighted-field@Glastopf@/kibana-highlighted-field@"
    ]
  },
  "sort": [
    1463752520507
  ]
}`

**Comment by t3chn0m4g3** [comment]
Seems to me like an issue with glastopf itself, have you checked [here](https://github.com/mushorg/glastopf)?

The tag `[ "_grokparsefailure" ]'` indicates that logstash was unable to parse the glastopf log message (rule does not match) so it will not be logged to Elastic.

**Comment by resech** [comment]
Good call, I had a further look at the glastopf code [here](https://github.com/mushorg/glastopf/blob/master/glastopf/modules/processing/profiler.py) found the following:

The original error message `(glastopf.modules.processing.profiler) An issue occured with the dedicated logger. Please check the configuration file.` is displayed when the glastopf logger is anything other than the Log Profiler. I'm surprised that message appears at all, since it looks like the Profiler is disabled in glastopf.cfg. 

It looks like setting 

`[profiler]
enabled = True
`

in glastopf.cfg appears to remedy the dedicated logger error and gives a little nicer grokparsefailure message:

`{
  "_index": "logstash-2016.05.23",
  "_type": "Glastopf",
  "_id": "AVTdgHGVM-u_eDo7F5nr",
  "_score": null,
  "_source": {
    "message": "2016-05-23 12:04:21,945 (glastopf.modules.reporting.auxiliary.log_profiler) Comment added to database for ip address 188.143.232.40",
    "@version": "1",
    "@timestamp": "2016-05-23T12:04:22.503Z",
    "path": "/data/glastopf/log/glastopf.log",
    "host": "f76e01c14c39",
    "type": "Glastopf",
    "tags": [
      "_grokparsefailure"
    ]
  },
  "fields": {
    "@timestamp": [
      1464005062503
    ]
  },
  "highlight": {
    "path": [
      "/data/@kibana-highlighted-field@glastopf@/kibana-highlighted-field@/log/glastopf.log"
    ],
    "type": [
      "@kibana-highlighted-field@Glastopf@/kibana-highlighted-field@"
    ]
  },
  "sort": [
    1464005062503
  ]
}`

Not sure if that's a change that is worth rolling up into the container though.

**Comment by t3chn0m4g3** [comment]
Thanks for the feedback.

Probably not, but I will keep the issue open and test if changing it breaks anything. If not I will commit it.
